devtools/maven/src/test/java/io/quarkus/maven/it/DevMojoIT.java

```diff
@@ -189,6 +189,34 @@ public void testThatTheApplicationIsReloadedOnKotlinChange() throws MavenInvocat
                 .atMost(1, TimeUnit.MINUTES).until(() -> getHttpResponse("/app/hello").contains("carambar"));
     }
 
+    @Test
+    public void testThatTheApplicationIsReloadedOnScalaChange() throws MavenInvocationException, IOException {
```

Comment:
By nature, these tests IT take some minutes  to run,  I wonder if this test should be a unit test using `QuarkusDevModeTest` since https://github.com/quarkusio/quarkus/pull/3317 has already landed. 

@gsmet WDYT?

Reply:
If `QuarkusDevModeTest` can cover this case, it would be ideal to use it. 
And if it does, then replacing the existing tests with something that uses `QuarkusDevModeTest` would be even better :)

Reply:
Sure,  I think tests that check if files have been correctly modified, deleted etc can leverage `QuarkusDevModeTest`. We have an issue for that?

Reply:
Not that I know of, but I haven't really checked